_includes/samples/VRF/RandomNumberConsumer.sol

```diff
@@ -18,24 +18,23 @@ contract RandomNumberConsumer is VRFConsumerBase {
 
     bytes32 internal keyHash;
     uint256 internal fee;
-
     uint256 public randomResult;
 
     /**
      * Constructor inherits VRFConsumerBase
      *
-     * Network: Rinkeby
-     * Chainlink VRF Coordinator address: 0xb3dCcb4Cf7a26f6cf6B120Cf5A73875B7BBc655B
-     * LINK token address: 0x01BE23585060835E02B77ef475b0Cc51aA1e0709
+     * Network: Goerli
+     * Chainlink VRF Coordinator address: 0x2Ca8E0C643bDe4C2E08ab1fA0da3401AdAD7734D
+     * LINK token address: 0x326C977E6efc84E512bB9C30f76E30c160eD06FB
      * Key Hash: 0x2ed0feb3e7fd2022120aa84fab1945545a9f2ffc9076fd6156fa96eaff4c1311
```

Comment:
Does this need to be 
`     * Key Hash: 0x79d3d8832d904592c0bf9818b621522c988bb8b0c05cdc3b15aea1b6e8db0c15`